src/libsync/owncloudpropagator.h

```diff
@@ -227,8 +240,20 @@ private slots:
         return next->scheduleSelfOrChild();
     }
 
+    void terminateJobs() {
+        // Send abort to the subjobs and emit abort finished signal
+        abortJobs();
```

Comment:
Hmm, maybe in fact I dont need it also... it will roll back to what was before this PR for the jobs which did not terminate within 5sec